src/wallet/wallet.cpp

```diff
@@ -4885,13 +4885,13 @@ CAmount CWalletTx::GetDebit(const isminefilter& filter) const
         return 0;
 
     CAmount debit = 0;
-    if(filter & ISMINE_SPENDABLE)
+    if(filter & ISMINE_SPENDABLE_ANY)
```

Comment:
Nit (while this line is being changed anyway):
```suggestion
    if (filter & ISMINE_SPENDABLE_ANY)
```